README.md

```diff
@@ -1,3 +1,7 @@
+# Warning
+|:warning: This fork is for _prototyping purpose only_. If you want to submit a PR to the Open Hospital project starts from the [official ISF (Informatici Senza Frontiere) repos](https://github.com/informatici).      |
+|:---------------------------|
+
```

Comment:
this part comes from Intesys but makes no sense in informatici repository, please remove it